lucien-ios/Protocols/Alertable.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Alertable.swift
+//  lucien-ios
+//
+//  Created by Ahmed, Guled on 10/9/17.
+//  Copyright © 2017 Intrepid Pursuits. All rights reserved.
+//
+
+import Foundation
+
+protocol Alertable {}
```

Comment:
I would consider renaming this. `Alertable` to me implies that something can be alerted - like it responds to a callback or something else similar. This is more like `AlertDisplaying`